src/modules/sovereign-clouds/cloud-options.ts

```diff
@@ -0,0 +1,50 @@
+import { IChoiceGroupOption } from '@fluentui/react';
+import { clouds, globalCloud } from '.';
+import { geLocale } from '../../appLocale';
+
+export class Sovereign {
+  protected profile = null;
+
+  constructor(profile: any) {
+    this.profile = profile;
+  }
+  public getOptions(): IChoiceGroupOption[] {
+    let options: IChoiceGroupOption[] = [];
+
+    clouds.forEach(cloud => {
+      options.push({
+        key: cloud.name,
+        text: cloud.name
+      });
+    });
+
+    options.unshift({
+      key: globalCloud.name,
+      text: globalCloud.name
+    });
+
+    if (!this.canAccessCanary()) {
+      options = options.filter(option => option.key !== 'Canary');
+    }
+
+    if (!this.canAccessChinaCloud()) {
+      options = options.filter(option => option.key !== 'China');
+    }
+    return options;
+  }
+
+  private canAccessChinaCloud() {
+    return geLocale === 'en-US';
```

Comment:
What is this for?

Reply:
I was using this when sovereign clouds in different countries was something to support. 

I would use  the locale to match the clouds I had. So that if a cloud for the region exists I would suggest to the user that they can opt in to using another cloud

Reply:
This specific line was tied specifically to the China cloud.. I used en_Us for demos

Reply:
I don't believe we can use locale to determine anything reliable regarding what clouds a user wants to be able to use.